src/assets/css/dashboard.css

```diff
@@ -409,9 +401,14 @@ div[data-role=controlgroup] a.ui-btn-active {
 }
 
 .transcodingProgress > div {
+    z-index: 900;
```

Comment:
This seems arbitrarily high.

Reply:
I can set it to an arbitrarily lower number if you want.

Reply:
The lower the better. By setting the z-index to a really high value then things like modals could be messed up if their z-index doesn't exceed the arbitrarily high value.